legal-api/src/legal_api/utils/legislation_datetime.py

```diff
@@ -41,6 +41,11 @@ def as_legislation_timezone(date_time: datetime) -> datetime:
         """Return a datetime adjusted to the legislation timezone."""
         return date_time.astimezone(pytz.timezone(current_app.config.get('LEGISLATIVE_TIMEZONE')))
 
+    @staticmethod
+    def as_legislation_timezone_from_date(_date: datetime.date) -> datetime.date:
+        """Return a datetime adjusted to the legislation timezone from a date object."""
+        return LegislationDatetime.as_legislation_timezone(datetime.datetime(_date.year, _date.month, _date.day))
```

Comment:
It seems that `as_legislation_timezone` doesn't make any difference in dev env. I'm going to change it to `gettz`